src/app/(sidebar)/account/fund/page.tsx

```diff
@@ -30,6 +35,15 @@ export default function FundAccount() {
     }
   }, [isError, isSuccess]);
 
+  if (!IS_TESTING_NETWORK) {
```

Comment:
I also tried out with adding a [middleware.ts](https://nextjs.org/docs/app/building-your-application/routing/middleware) to reroute to `404` if `nextUrl` includes `mainnet` in params. However, it doesn't always work in all cases. ex. user directly types in `/account/fund` in a browser which doesn't populate params until after user lands on the page.

another method i tried is `router.push` on the client side but that produced a glitch which wasn't a good ux.

so I came up with this.

Reply:
I'll update the copy in the next pr when I make the overall changes following the updates